src/mscorlib/GenerateSplitStringResources.Unix.targets

```diff
@@ -0,0 +1,31 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PrepareResourcesDependsOn>GenerateSplitStringResources;$(PrepareResourcesDependsOn)</PrepareResourcesDependsOn>
+  </PropertyGroup>
+    
+  <Target Name="GenerateSplitStringResources"
+          Inputs="$(SplitTextStringResource)"
+          Outputs="@(SplitTextStringResource->'$(IntermediateOutputPath)%(Filename).txt')">
+
+    <ItemGroup>         
+      <Internal_ResGenDefines Remove="" />
+      <Internal_ResGenDefines Include="INCLUDE_DEBUG" />
+      <Internal_ResGenDefines Include="INCLUDE_RUNTIME" />
+      <Internal_ResGenDefines Include="%(SplitTextStringResource.ResGenDefines)" />
+    </ItemGroup>    
+
+    <Exec Command="python &quot;$(MSBuildThisFileDirectory)/Tools/PreprocessResources/ResPreprocess.py&quot; &quot;%(SplitTextStringResource.Identity)&quot; &quot;$(IntermediateOutputPath)%(SplitTextStringResource.Filename).txt&quot; @(Internal_ResGenDefines->'-D:%(Identity)', ' ')" />
```

Comment:
@ellismg, can we define it overrideable like:

```xml
<PropertyGroup>
    <PYTHON_EXE Condition="'$(PYTHON_EXE)'==''">python</PYTHON_EXE>
    ...
</PropertyGroup>

<Exec Command="$(PYTHON_EXE) .." /> 
```

so we will set `PYTHON_EXE` env variable.